src/test/java/scenario/ScenarioSteps.java

```diff
@@ -183,6 +183,18 @@ public ScenarioSteps() throws IOException {
                     }
                 });
 
+	    Given("I have a gateway as user {word} using the {word} connection profile without a Wallet",
```

Comment:
The user name argument isn't being used. The (currently) only supported value could either be inserted directly into the stepdef text, or the value of the `userName` argument should probably be checked that it matches the expected value in the code.

Reply:
Right. Can I rephrase it to "I have a gateway with identity User1 using the {word} connection profile"?

Reply:
Yes, that looks fine. Just remember to change the accompanying function signature to just have the one (String tlsType) parameter.